src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1315,6 +1315,7 @@ private static bool IsMemberDeclaration(SyntaxNode node)
                 case SyntaxKind.PropertyDeclaration:
                 case SyntaxKind.GetAccessorDeclaration:
                 case SyntaxKind.SetAccessorDeclaration:
+                case SyntaxKind.InitAccessorDeclaration:
```

Comment:
I don't think it's a good idea to update this one without knowing the change it introduce.

This **whole** method isn't covered in the tests. See how https://github.com/dotnet/roslyn/pull/48158 is green.